arch/arm/soc/st_stm32/stm32f2/Kconfig.defconfig.stm32f207xe

```diff
@@ -0,0 +1,34 @@
+# Kconfig - ST Microelectronics stm32f207XE MCU
```

Comment:
Change commit message to something less personal:
Add STM32F207XE SoC. Used and tested (provide list of tests) on out of tree custom board.